test/components/game/Game.test.js

```diff
@@ -141,33 +173,32 @@ it('does not allow user to pick a cell if game has reached a terminal state', ()
 });
 
 it('random computer player marks final position', () => {
-  const wrapper = Enzyme.mount(
-    <Game board={Array(9).fill(EMPTY)} gameMode={gameMode.HUMANVSRANDOM} />,
-  );
+  jest.useFakeTimers();
+  const wrapper = Enzyme.mount(humanVsRandomGame());
   wrapper.setState({
     grid: ['X', 'O', 'X', 'O', 'O', EMPTY, 'X', 'X', EMPTY],
   });
   wrapper
     .find(Cell)
     .at(8)
     .simulate('click');
-  expect(wrapper.state('grid')).toEqual([
-    'X',
-    'O',
-    'X',
-    'O',
-    'O',
-    'X',
-    'X',
-    'X',
-    'O',
-  ]);
+  setTimeout(() => {
```

Comment:
How is this working? Do we need the timeout in the test, if `jest.useFakeTimers()` is stubbing out the timeout for the computer player?